lib/plaid/account.rb

```diff
@@ -1,5 +1,128 @@
 module Plaid
-  # Public: Representation of account data.
+  # Public: Representation of user account data.
   class Account
+    # Public: The String unique ID of the account. E.g.
+    # "QPO8Jo8vdDHMepg41PBwckXm4KdK1yUdmXOwK".
+    attr_reader :id
+
+    # Public: The String account ID unique to the accounts of a particular
+    # access token. E.g. "KdDjmojBERUKx3JkDd9RuxA5EvejA4SENO4AA".
+    attr_reader :item_id
+
+    # Public: The Float value of the current balance for the account.
+    attr_reader :current_balance
+
+    # Public: The Float value of the available balance for the account.
+    #
+    # The Available Balance is the Current Balance less any outstanding holds
+    # or debits that have not yet posted to the account. Note that not all
+    # institutions calculate the Available Balance. In the case that Available
+    # Balance is unavailable from the institution, Plaid will either return an
+    # Available Balance value of null or only return a Current Balance.
+    attr_reader :available_balance
+
+    # Public: The Symbol institution type, e.g. :wells.
+    attr_reader :institution
+
+    # Public: The Hash with additional information pertaining to the account
+    # such as the limit, name, or last few digits of the account number. E.g.
+    # {"name": "Plaid Savings", "number": "9606" }.
+    attr_reader :meta
+
+    # Public: The String account name. Obtained from the #meta.
+    attr_reader :name
```

Comment:
Well, there's no actual duplication in memory, because `@name` is just a reference to the same string. Here's a proof:

```
irb(main):005:0> attrs = { meta: { name: 'test' } }
=> {:meta=>{:name=>"test"}}
irb(main):006:0> name = attrs[:meta][:name]
=> "test"
irb(main):007:0> name.object_id
=> 70294883716640
irb(main):008:0> attrs[:meta][:name]
=> "test"
irb(main):009:0> attrs[:meta][:name].object_id
=> 70294883716640
```
The object_id is the same.